microprofile-metrics/src/test/java/org/jboss/eap/qe/microprofile/metrics/integration/config/CustomMetricSystemPropertyTest.java

```diff
@@ -46,11 +47,13 @@ public static WebArchive createDeployment() {
     }
 
     void setConfigProperties(int increment) throws IOException, ConfigurationException, TimeoutException, InterruptedException {
-        Assert.assertEquals(ClientConstants.SUCCESS, managementClient.getControllerClient()
-                .execute(Util.getWriteAttributeOperation(SYSTEM_PROPERTY_ADDRESS, VALUE, increment))
-                .get(ClientConstants.OUTCOME)
-                .asString());
-        new Administration(ManagementClientProvider.onlineStandalone()).reload();
+        try (OnlineManagementClient client = ManagementClientProvider.onlineStandalone()) {
+            Assert.assertEquals(ClientConstants.SUCCESS,
```

Comment:
AFAIK you can just call `assertSuccess()` on `client.execute` return value.

Reply:
nice catch, fixed in all occurrences in this PR

fixed